src/strings/unicode_ops.c

```diff
@@ -417,7 +417,7 @@ void MVM_unicode_release(MVMThreadContext *tc)
 /* Looks up a grapheme by name. Lazily constructs a hash. */
 MVMString * MVM_unicode_string_from_name(MVMThreadContext *tc, MVMString *name) {
     MVMuint64 size;
-    char *cname = MVM_string_ascii_encode(tc, name, &size, 0);
+    char *cname = MVM_string_ascii_encode(tc, MVM_string_uc(tc, name), &size, 0);
```

Comment:
`MVM_string_uc` will allocate a new `MVMString`, which in turn may cause the pointer to `name` to become outdated. Thus it is not safe to use it again, however it is used again below in the call to `MVM_unicode_lookup_by_name`. This may cause occasional reading of junk memory and SEGVs.

One solution would be to do an `MVMROOT` of name. However, I note that we currently will end up doing the `uc` and ASCII encoding twice, which is a bit wasteful. A neater and more efficient fix might be to extract the part of the looking up of codepoint names by a C string into a function, and call that from here, so that we care only doing the uc/encode one time.